source/playground/src/editor.tsx

```diff
@@ -128,6 +129,7 @@ export function Editor(props: {
     const config = {
       sources: [["code", code]] as [string, string][],
       languageFeatures: [],
+      profile: await getTargetProfileFromEntryPoint("main.qs", code),
```

Comment:
To get the "unrestricted" by default behavior, couldn't this playground specific code just use "unrestricted" if it didn't find the profile explicitly?
```suggestion
      profile: await getTargetProfileFromEntryPoint("main.qs", code) ?? "unrestricted",
```

Reply:
I guess it means that, because the default is different for different actions, you could be looking at the playground, see the generated qir based off of an adaptive_rif compilation, and click "Run" triggering a compilation based off of unrestricted profile. If you haven't specified a profile in the EntryPoint. In VS Code that makes more sense, but maybe in the Playground it makes more sense to always default to unrestricted? Though, in that case you'd _have_ to specify the profile in the EntryPoint before you'd be able to see the qir or rir.